profiles/iperf/profile.mk

```diff
@@ -1,3 +1,4 @@
 
-WORLD_SUBDIRS_ZROUTER+=	target/sbin/iperf
+#WORLD_SUBDIRS_ZROUTER+=	target/sbin/iperf
```

Comment:
if you move it, why leave old point under comment?
To have error when uncommented? :)